lib/grizzly/zwave/decoder.ex

```diff
@@ -52,7 +53,13 @@ defmodule Grizzly.ZWave.Decoder do
                 ),
                 do: decode(unquote(command_module), params)
           end
-        end
+        end ++
+          [
+            quote do
+              # No Operation (0x00) - There is no command byte or args for this command, only the command class byte
+              def from_binary(<<0x00>>), do: decode(Commands.NoOperation, [])
```

Comment:
All this really good! Since this is a special command I am okay with handling it with a match like this, however, to avoid adding to the macro can just stick this line (line 60) right below line 66: https://github.com/smartrent/grizzly/blob/f3749a084ad46846ff372901e349933c1c53b53e/lib/grizzly/zwave/decoder.ex#L66

```
unquote(from_binary)
def from_binary(<<0x00>>), do: decode(Commands.NoOperation, [])
```

Since we are just generating code and not providing checks during compile time, not adding to the macro is okay and I find it easier to understand/maintain.

Reply:
Yep. Fixed.